src/vcpkg/dependencies.cpp

```diff
@@ -1185,14 +1147,8 @@ namespace vcpkg
         std::sort(excluded.begin(), excluded.end(), &InstallPlanAction::compare_by_name);
 
         static auto actions_to_output_string = [&](const std::vector<const InstallPlanAction*>& v) {
-            return Strings::join("\n", v, [&](const InstallPlanAction* p) {
-                return to_output_string(p->request_type,
-                                        p->displayname(),
-                                        p->build_options,
-                                        p->source_control_file_and_location.get(),
-                                        p->installed_package.get(),
-                                        builtin_ports_dir);
-            });
+            return Strings::join(
+                "\n", v, [&](const InstallPlanAction* p) { return to_output_string(*p, builtin_ports_dir); });
```

Comment:
As an aside the name 'to_output_string' is sadly not amazing here. Maybe `InstallPlanAction::to_plan_text()` should be a thing?